### PR TITLE
Add org.fcitx.Fcitx5.Addon.McBopomofo

### PR DESCRIPTION
**Pull request by lukhnos**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

Note: fcitx5-mcbopomofo is a third-party add-on to fcitx5, an input method editor (IME) framework. I'm the maintainer of fcitx5-mcbopomofo. The PR is modelled after https://github.com/flathub/flathub/pull/3311. Our project is also distributed on other *NIX, for example: https://www.freshports.org/chinese/fcitx5-mcbopomofo/                                                                                                                                       
                                                                                                                                                         
Thank you!

### CODE REVIEW

**Comment by lukhnos** [comment]
bot, build org.fcitx.Fcitx5.Addon.McBopomofo

**Comment by lukhnos** [comment]
bot, build org.fcitx.Fcitx5.Addon.McBopomofo

File: org.fcitx.Fcitx5.Addon.McBopomofo.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DFMT_TEST=Off
+      - -DBUILD_SHARED_LIBS=ON
+    sources:
+      - type: git
+        url: https://github.com/fmtlib/fmt
+        tag: 9.1.0
+  - name: fcitx5-mcbopomofo
+    buildsystem: cmake
```
Comment (hfiguiere) [inline org.fcitx.Fcitx5.Addon.McBopomofo.yaml:30]: does cmake-ninja works? if it does it's better.

**Comment by lukhnos** [comment]
bot, build org.fcitx.Fcitx5.Addon.McBopomofo

File: org.fcitx.Fcitx5.Addon.McBopomofo.yaml (hunk 1 above)
Comment (lukhnos) [reply org.fcitx.Fcitx5.Addon.McBopomofo.yaml]: It does. Updated now.

**Pull request closed without merging**